src/test/java/com/expediagroup/hiveberg/TestInputFormatWithHadoopCatalog.java

```diff
@@ -189,15 +181,6 @@ public void testGetSplitsNoCatalog() throws IOException {
     format.getSplits(conf, 1);
   }
 
-  @Test(expected = IOException.class)
-  public void testGetSplitsInvalidWarehouseLocationUri() throws IOException {
```

Comment:
Is this test covered in whatever is testing `pathAsURI`?

Reply:
Ping ^

Reply:
Ah yes sorry, a similar answer to above, I've removed it because we're not using the warehouse location property anymore, but we still have all the tests for the other properties we're using